web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -333,7 +326,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
 
             Portal.data.ActiveGeoNetworkRecordStore.instance().add(record);
         }
-        Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
+
+        if (!multiSelect) {
+            Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
```

Comment:
On a related note, what do you think about making the following changes to renaming `ACTIVE_GEONETWORK_RECORD_ADDED` simply to `COLLECTION_ADDED` and so on? Bit shorter and more descriptive I think.